subgraph/subgraph.yaml

```
@@ -27,12 +27,13 @@ dataSources:
         - TokenRoleGranted
         - TokenRoleRevoked
         - Transfer
+        - CollectionOwner
         - Token
         - Owner
         - Controller
       abis:
         - name: FleekNFA
-          file: ./abis/FleekNFA.json
+          file: ../deployments/mumbai/FleekERC721.json
```

In another PR we put artifacts, and here it's deployment.  I actually think both are useful depending on what you are doing so maybe we add some docu like this:

```suggestion
          # Use this if you want to test against last deployed contracts
          #file: ../contracts/deployments/mumbai/FleekERC721.json
          # Use this if you want to test local contract changes
          file: ../contracts/artifacts/contracts/FleekERC721.sol/FleekERC721.json
```

---

I agree with adding a comment there.